CHANGELOG.md

```diff
@@ -17,6 +18,89 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent alpha or release-candidate API versions, the delta with respect to the previous pre-release
 * for a public API version, the consolidated changes since the release of the previous public API version
 
+# r1.3
+## Release Notes
+
+This release contains the definition and documentation of
+* device-roaming-status v0.6.0
```

Comment:
Just checking, will it be on 0.6.0 where others on 0.6.1

Reply:
There wasn't any change since r1.2. The APIs are independent, so the numbering will run separately.